analytic_diffuse/models.py

```diff
@@ -121,14 +168,20 @@ def gauss(phi, theta, a, el0vec=None):
     el0vec: ndarray of float
         Cartesian vector describing lmn displacement of Gaussian center, shape (3,).
     """
+    if phi is None:
+        phi = np.zeros_like(theta)
+
     if el0vec is None:
         el0vec = np.array([0, 0, 0])
 
     sigsq = a**2 / (2 * np.pi)
     lmn = _angle_to_lmn(phi, theta)
-    disp = (lmn - el0vec)[:, :2]  # Only l an m
+    disp = (lmn - el0vec)[:, :2]  # Only l and m
     ang = np.linalg.norm(disp, axis=1)
-    return np.exp(-ang**2 / (2 * sigsq))
+    try:
+        return np.exp(-ang**2 / (2 * sigsq))
+    except AttributeError:
```

Comment:
Why would it throw an AttributeError?

Reply:
Yeah this is a bit opaque. As it turns out, if `ang` is an mpmath `mpf` object, `numpy` throws an attribute error when it tries to apply any of its functions to it. That said, I'm not sure this is the _quickest_ way to do this. And it doesn't work in reverse -- if you _want_ the high-precision version, but `ang` is a float, it'll still use `numpy`. 

This works for the integration routine, which makes `ang` an mpf when called with high-precision.